include/sdsl/config.hpp

```diff
@@ -106,16 +106,16 @@ struct key_bwt_trait_impl<8,T> {
 	static const char* KEY_BWT;
 };
 
-template<typename T>
+template <typename T>
```

Comment:
Can you remove this file from the commit? Since there are no changes in this file necessary, I suggest not fixing formatting issues here.